src/test/java/com/google/gcp_variant_transforms/entity/VariantTest.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 Google LLC
+
+package com.google.gcp_variant_transforms.entity;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import org.junit.Test;
```

Comment:
Use Truth instead, for consistency.

assertThat().matches().
assertThat(...).isTrue().
and so on, throughout the tests.

Please check HeaderIteratorTest for reference, or this tutorial - https://www.baeldung.com/google-truth .

Reply:
Resolved.